### PR TITLE
docs/FAQ: Clean up entries regarding SSH settings

### PR DESCRIPTION
**Pull request by mrichtarsky**
Link to recommended ones

### CODE REVIEW

File: docs/faq.rst [hunk 1]
```diff
@@ -602,31 +602,22 @@ You will also get this error if you try to access a repository with a key that u
...
-        # Client kills connection after 3*30 seconds without server response:
-        ServerAliveInterval 30
-        ServerAliveCountMax 3
+Why does Borg disconnect or hang when backing up to a remote server?
+--------------------------------------------------------------------
 
-You can also do the opposite and configure SSH on server side in
-/etc/ssh/sshd_config, to make the server send keep-alive requests to the client:
+Communication with the remote server (using an ssh: repo URL) happens via the SSH
+protocol. This can lead to some issues that would not occur during a local backup:
```
Comment (ThomasWaldmann) [inline docs/faq.rst:611]: ... via a ssh connection.

(it's not like borg is talking ssh protocol somehow, there is a pair of ssh-sshd processes doing that)

File: docs/faq.rst [hunk 2]
```diff
@@ -602,31 +602,22 @@ You will also get this error if you try to access a repository with a key that u
...
 
-::
+- Since Borg does not send data all the time, the connection may get closed, leading
+  to errors like "connection closed by remote".
```
Comment (ThomasWaldmann) [inline docs/faq.rst:614]: Not sure if that issue matches that error msg.

Comment (mrichtarsky) [reply docs/faq.rst:614]: That was the message from the original FAQ entry, that's why I kept it. In my setup, there was a different one but I don't have it anymore (also, I used the `socat` pull mode so not sure if my error message would have been representative anyway)

**Comment by mrichtarsky** [comment]
@ThomasWaldmann  Could you please merge so I can downport?

File: docs/faq.rst [hunk 3]
```diff
@@ -642,32 +633,10 @@ could try to work around:
...
-    # /etc/ssh/sshd_config on borg repo server - kill connection to client
-    # after ClientAliveCountMax * ClientAliveInterval seconds with no response
-    ClientAliveInterval 20
-    ClientAliveCountMax 3
-
-If you have multiple borg create ... ; borg create ... commands in a already
-serialized way in a single script, you need to give them ``--lock-wait N`` (with N
-being a bit more than the time the server needs to terminate broken down
-connections and release the lock).
+Please see :ref:`disconnect_hang`
```
Comment (ThomasWaldmann) [inline docs/faq.rst:639]: can you move this to right above the disconnect_hang label?

maybe one could just say "see below" then

**Pull request merged**